spk/transmission/src/service-setup.sh

```diff
@@ -2,22 +2,22 @@
 # This gives tranmission the power to execute python scripts on completion (like TorrentToMedia).
 PYTHON_DIR="/usr/local/python"
 PATH="${SYNOPKG_PKGDEST}/bin:${PYTHON_DIR}/bin:${PATH}"
```

Comment:
As there is no `SPK_DEPENDS=python...` in this package, it is not defined which python should be used. 

For synocommunity packages the path `/usr/local/{package}` must not be used anymore, i.e. it must be defined as `/var/packages/{package}/target`.

I suppose that python2 should not be used anymore, so the package must be  `python3` or `python38`.

Proposal:
remove these lines and rely on python that is already installed and available in the path.

if no python is found in path, a lookup for synocommunity python3 and python38 could be used here to add it to the path for transmission.

Reply:
Thanks. Good catch!

Reply:
As of DSM 7 Python 3 is standard available and can be found via the path.

Reply:
>if no python is found in path, a lookup for synocommunity python3 and python38 could be used here to add it to the path for transmission.

@hgy59 I didn't do any fancy detection because that is what $PATH is for anyway. Don't need to do it twice and this way it still works if a python package is added or removed. Agreed, for DSM7 the build in Python should be good enough.